consensus/istanbul/backend/announce.go

```diff
@@ -587,8 +559,16 @@ func (sb *Backend) answerAnnounceMsg(address common.Address, node *enode.Node, v
 // message. This is to force all validators that send an announce message to
 // create as succint message as possible, and prevent any possible network DOS attacks
 // via extremely large announce message.
-func (sb *Backend) validateAnnounce(announceData *announceData) (bool, error) {
-	logger := sb.logger.New("func", "validateAnnounce")
+func (sb *Backend) validateAnnounce(msgAddress common.Address, announceData *announceData) (bool, error) {
+	logger := sb.logger.New("func", "validateAnnounce", "msg address", msgAddress)
+
+	// Ensure the version in the announceData is not older than the signed
+	// announce version known by this node
+	knownVersion, err := sb.signedAnnounceVersionTable.GetVersion(msgAddress)
+	if err == nil && knownVersion < announceData.Version {
+		logger.Info("Announce message has version older than known version from signed announce table", "msg version", announceData.Version, "known version", knownVersion)
```

Comment:
Log level should not be info. If we believe this is a problematic case, it should be warning or if it is not, it should be debug.